src/metrics/metrics.cc

```diff
@@ -64,56 +81,32 @@ prometheus::Family<prometheus::Counter> &setup_family(
 	return prometheus::BuildCounter().Name(name).Help(help).Register(*registry);
 }
 
-class Counters {
+class PrometheusMetrics {
 public:
-	Counters() {
-		registry = std::make_shared<prometheus::Registry>();
-		// NOLINTBEGIN(cppcoreguidelines-prefer-member-initializer)
-		packet_counter = &setup_family(
-		    "observed_packets_total_client",
-		    "Number of observed packets from and for client", registry);
-		// NOLINTEND(cppcoreguidelines-prefer-member-initializer)
-
-		// A very hacky way to allow compile time checking if metrics have been
-		// set. Any enum value not used will throw a compile time error.
-		Counter::Key start = Counter::KEY_START;
-		switch (start) {
-		case Counter::KEY_START:
-		case Counter::CLIENT_RX_PACKETS:
-			counters[Counter::CLIENT_RX_PACKETS] = Counter(
-			    {{"protocol", "tcp"}, {"direction", "rx"}}, packet_counter);
-			[[fallthrough]];
-		case Counter::CLIENT_TX_PACKETS:
-			counters[Counter::CLIENT_TX_PACKETS] = Counter(
-			    {{"protocol", "tcp"}, {"direction", "tx"}}, packet_counter);
-		case Counter::KEY_END:
-			break;
-		}
+	PrometheusMetrics()
+	    :
+	      registry(std::make_shared<prometheus::Registry>()) {
+		master = master::Master(registry);
 	}
 
 	std::shared_ptr<prometheus::Registry> get_registry() {
 		return registry;
 	}
 
-	inline Counter& get(Counter::Key type) {
-		// Completely safe, as all values are defined
-		return counters[type]; // NOLINT(cppcoreguidelines-pro-bounds-constant-array-index)
-	}
+	// Master metrics
+	master::Master master; // NOLINT(cppcoreguidelines-non-private-member-variables-in-classes)
 
 private:
-	// Registry, used by packet_counter and counters
+	// Registry
 	std::shared_ptr<prometheus::Registry> registry;
-
-	// Metric(s)
-	prometheus::Family<prometheus::Counter> *packet_counter;
-
-	// All counters
-	std::array<Counter, Counter::Key::KEY_END + 1> counters;
 };
-Counters counters;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
+PrometheusMetrics prom_metrics;  // NOLINT(cppcoreguidelines-avoid-non-const-global-variables)
```

Comment:
Why not `prometheus_metrics` instead of `prom_metrics`? 
I think is more clear.